qa/integration-tests/src/test/java/io/camunda/zeebe/it/clustering/ClusteredSnapshotTest.java

```diff
@@ -97,7 +97,13 @@ void shouldSendSnapshotOnReconnect(final Consumer<ClusteringRule> snapshotTrigge
     final var adminService =
         clusteringRule.getBroker(leaderId).getBrokerContext().getBrokerAdminService();
     ControllableExporter.updatePosition(true);
-    clusteringRule.fillSegment();
+    clusteringRule.fillSegments(
+        2,
```

Comment:
🔧 this was 1 before, right? given the second parameter taking care of the threshold being exceeded, we could keep `minimumSegmentCount` it at 1 as well?

Reply:
Not really, we changed `<=` to `<` in `fillSegments`. Before and after, we'd wait until at least two segments are created which makes sense since we usually want at least two segments so that one can get deleted by log compaction.